README.md

```diff
@@ -1,24 +1,74 @@
-# README
+# Wallet app
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Wallet app will be a classic example of a budget and expenses website. I had create a fully functional digital wallet that will show the list of groups and operations that you have.
 
-Things you may want to cover:
+## Video presentation
+[video](https://www.loom.com/share/1eded00cafcf4442a9e91433420101f9)
+
+## Live Demo
+
+[Demo](https://arturo-wallet.herokuapp.com/)
+
+## Built With
 
 * Ruby version
+- ROR
+- Postgresql
+- Rspec
 
-* System dependencies
+## Getting Started
 
 * Configuration
+1. Open [Repo-dev branch](https://github.com/Artbsc1992/wallet)
+2. Open VSCode or terminal and navigate to the folder where you would like to clone the project.
+3. Type "pwd" to see your working directory.
+4. [clone repository](https://github.com/Artbsc1992/wallet)
+
+```bash
+$ git clone https://github.com/Artbsc1992/wallet
+```
+
+5. Install gem's by running
+
+```bash
+bundle install
+```
+
+6. Start server by running
+
+```bash
+rails s
+```
+
+### Prerequisites
+
+- rails
+- ruby
+- postgresql
+
+
+## Authors
+
+👤 **Arturo Hermida**
+
+- GitHub: [@Artbsc1992](https://github.com/Artbsc1992)
+- Twitter: [@Arturo_D_Rock](https://twitter.com/Arturo_D_Rock)
+- LinkedIn: [Arturo Hermida](https://www.linkedin.com/in/arturo-hermida29/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
 
-* Database creation
+## Show your support
 
-* Database initialization
+Give a ⭐️ if you like this project!
 
-* How to run the test suite
+## Acknowledgments
 
-* Services (job queues, cache servers, search engines, etc.)
+- Hat tip to all our code reviewers and partners at Microverse.
```

Comment:
in the acknowledgement section please give credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/). 👍🏾 💯